Sources/RealModule/ApproximateEquality.swift

```diff
@@ -9,103 +9,130 @@
+  /// useful comparison is recoverable, so we simply do not try; infinity compares
+  /// not equal to any finite value with any allowed tolerance.
+  @inlinable @inline(__always)
+  public func isApproximatelyEqual(
+    to other: Self,
+    relativeTolerance: Magnitude
+  ) -> Bool {
+    return isApproximatelyEqual(
+      to: other,
+      relativeTolerance: relativeTolerance,
+      absoluteTolerance: relativeTolerance * Magnitude.leastNormalMagnitude
+    )
+  }
+  
+  /// Compares `self` and `other` for approximate equality with a specified tolerances.
   ///
-  /// - `.relative(t)`
-  ///   `a` and `b` are approximately equal with relative tolerance `t` if
-  ///   ```
-  ///   let scale = max(a.magnitude, b.magnitude, .leastNormalMagnitude)
-  ///   return (a - b).magnitude < t * scale
-  ///   ```
-  ///   This type of tolerance matches well with the scale invariance of floating-point,
-  ///   making it a good choice for most problems. If you do not specify a tolerance,
-  ///   `approximatelyEquals` uses a relative tolerance of `sqrt(ulpOfOne)`.
+  /// `a.isApproximatelyEqual(to: b, relativeTolerance: rtol, absoluteTolerance: atol)`
+  /// is `true` if a and b are equal, or if they are finite and
+  /// ```
+  /// |a - b| ≤ max(atol, rtol * max(|a|,|b|))
+  /// ```
   ///
-  ///   The tolerance `t` must be in `.ulpOfOne ..< 1`.
+  /// Note that if `relativeTolerance` is omitted or is zero, a pure absolute tolerance is used:
+  /// ```
+  /// |a - b| ≤ atol
+  /// ```
   ///
-  /// - `.relative(t, minimumScale: s)`
-  ///   In some cases it is desirable to override the `minimumScale` used in a relative
-  ///   comparison (either by setting it to zero, so that there is no rolloff at zero, or by setting it
-  ///   to a much larger value to account for cancellation). When a `minimumScale` is
-  ///   specified, the computation performed is:
-  ///   ```
-  ///   let scale = max(a.magnitude, b.magnitude, minimumScale)
-  ///   return (a - b).magnitude < t * scale
-  ///   ```
-  ///   The tolerance `t` must be in `.ulpOfOne ..< 1`.  The `minimumScale` must
-  ///   be non-negative and finite.
+  /// If `absoluteTolerance` is zero, a pure relative tolerance is used:
+  /// ```
+  /// |a - b| ≤ rtol * max(|a|,|b|)
+  /// ```
   ///
-  /// - Parameters:
-  ///   - other: The value to compare.
-  ///   - tolerance: Either `.relative(...)` or `.absolute(t)`.
+  /// Note that if `absoluteTolerance` is _omitted_,
+  /// `relativeTolerance * .leastNormalMagnitude` is used, _not zero_.
   @inlinable
-  public func approximatelyEquals(
-    _ other: Self,
-    tolerance: Tolerance<Magnitude> = .relative()
+  public func isApproximatelyEqual(
+    to other: Self,
+    relativeTolerance: Magnitude = 0,
+    absoluteTolerance: Magnitude
   ) -> Bool {
-    // If a and b are actually equal, then they are certainly *almost* equal,
-    // with any allowable tolerance.
+    assert(absoluteTolerance >= 0 && absoluteTolerance.isFinite)
```

Comment:
```suggestion
    precondition(absoluteTolerance >= 0 && absoluteTolerance.isFinite)
```

Reply:
These are deliberately asserts rather than preconditions; these are not performing a test that's required for safety--the result is perfectly well-defined even when out-of-range tolerances are used, and we don't want to have them in release builds.

Reply:
Makes sense. I think it’d be cool if there were some comment to that effect.